ssl/statem/statem.c

```diff
@@ -174,8 +175,7 @@ int ossl_statem_accept(SSL *s)
     return state_machine(s, 1);
 }
 
-static void (*get_callback(SSL *s))(const SSL *, int, int)
-{
+static void (*get_callback(SSL *s)) (const SSL *, int, int) {
```

Comment:
If you introduce a typedef (which should be done anyway I think) the problem will be fixed.